src/Microsoft.Health.Dicom.Functions/Indexing/ReindexDurableFunction.Orchestration.cs

```diff
@@ -0,0 +1,124 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+using Microsoft.Health.Dicom.Core.Features.Indexing;
+using Microsoft.Health.Dicom.Functions.Indexing.Models;
+
+namespace Microsoft.Health.Dicom.Functions.Indexing
+{
+    public partial class ReindexDurableFunction
+    {
+        /// <summary>
+        ///  The orchestration function to add extended query tags.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="logger">The logger.</param>
+        /// <returns>The task.</returns>
+        [FunctionName(nameof(AddAndReindexTagsAsync))]
+        public async Task AddAndReindexTagsAsync(
+            [OrchestrationTrigger] IDurableOrchestrationContext context,
+            ILogger logger)
+        {
+            EnsureArg.IsNotNull(context, nameof(context));
+            logger = context.CreateReplaySafeLogger(logger);
+            var input = context.GetInput<IReadOnlyList<AddExtendedQueryTagEntry>>();
+            var storeEntires = await context.CallActivityAsync<IReadOnlyList<ExtendedQueryTagStoreEntry>>(
+                nameof(AddTagsAsync),
+                 input);
+            ReindexOperation reindexOperation = await context.CallActivityAsync<ReindexOperation>(
+                 nameof(PrepareReindexingTagsAsync),
+                 new PrepareReindexingTagsInput { OperationId = context.InstanceId, TagKeys = storeEntires.Select(x => x.Key).ToList() });
+            await context.CallSubOrchestratorAsync(nameof(ReindexTagsAsync), input: reindexOperation);
+        }
+
+        /// <summary>
+        /// The orchestration function to Reindex Extended Query Tags.
+        /// </summary>
+        /// <param name="context">the context.</param>
+        /// <param name="logger">The logger.</param>
+        /// <returns></returns>
+        [FunctionName(nameof(ReindexTagsAsync))]
+        public async Task ReindexTagsAsync([OrchestrationTrigger] IDurableOrchestrationContext context,
```

Comment:
nit: put `[OrchestrationTrigger] IDurableOrchestrationContext context,` on the next line